docs/installation.rst

```diff
@@ -70,6 +70,12 @@ You can use the following directives in your settings file:
     your own permission code. Default is ``None`` which means
     everybody can execute export action.
 
+``IMPORT_EXPORT_DIFF_BY_WORDS``
```

Comment:
I think this would be better as `IMPORT_EXPORT_DIFF_BY_CHARS`. This is because in your example `12345` is not a word, its a number.

Reply:
Ok, I will fix it.